test/e2e/run_test.go

```diff
@@ -1790,4 +1790,11 @@ WORKDIR /madethis`, BB)
 		_, err = strconv.Atoi(containerPID) // Make sure it's a proper integer
 		Expect(err).To(BeNil())
 	})
+
+	It("podman run check personality support", func() {
+		session := podmanTest.Podman([]string{"run", "--personality=LINUX32", "--name=testpersonality", ALPINE, "uname", "-a"})
```

Comment:
do we need to skip this test at the moment when runc is used?

Reply:
@giuseppe oh yes thanks for spotting this. Adding a skip.